lib/puppet/vendor/rgen/lib/rgen/metamodel_builder/builder_extensions.rb

```diff
@@ -395,7 +395,7 @@ def get<%= firstToUpper(name) %>
     

```

Comment:
Never modify the vendored version directly! Users can install other versions of a vendored library - they are not forced to use what we ship. Patching has to be done through normal coding or worst case monkey patching just as if the library was not vendored.